src/Servers/HttpSys/src/RequestProcessing/ResponseBody.cs

```diff
@@ -210,7 +210,7 @@ private List<GCHandle> PinDataBuffers(bool endOfRequest, ArraySegment<byte> data
             else if (!hasData && !addTrailers)
             {
                 // No data
-                dataChunks = new HttpApiTypes.HTTP_DATA_CHUNK[0];
+                dataChunks = Array.Empty<HttpApiTypes.HTTP_DATA_CHUNK>();
```

Comment:
This will be pinned and passed to native code, are there any implications with using the global empty array in that case?

Reply:
@Tratcher, thoughts?

Reply:
It's probably better to pin a singleton like this than an object in gen0. The pin for these is usually very short anyways.